packages/cli/src/commands/cat.js

```diff
@@ -4,10 +4,12 @@ import os from 'os';
 import { E } from '@endo/far';
 import { makeRefReader } from '@endo/daemon';
 import { withEndoAgent } from '../context.js';
+import { parsePetNamePath } from '../pet-name.js';
 
 export const cat = async ({ name, agentNames }) =>
   withEndoAgent(agentNames, { os, process }, async ({ agent }) => {
-    const readable = await E(agent).lookup(name);
+    const namePath = name && parsePetNamePath(name);
+    const readable = await E(agent).lookup(...namePath);
```

Comment:
`name` is required for the `cat` command:
```suggestion
    const readable = await E(agent).lookup(...parsePetNamePath(name));
```